src/lazy/expanded/macro_table.rs

```diff
@@ -188,43 +189,69 @@ impl Default for MacroTable {
 }
 
 impl MacroTable {
-    pub const SYSTEM_MACRO_KINDS: &'static [MacroKind] = &[
-        MacroKind::None,
-        MacroKind::ExprGroup,
-        MacroKind::MakeString,
-        MacroKind::MakeSExp,
-        MacroKind::Annotate,
-    ];
```

Comment:
🪧 This was a holdover from before we had a proper system macro table. It's no longer in use.